truelearn/learning/_ink_classifier.py

```diff
@@ -174,9 +174,9 @@ def __eval_matching_quality(
             bias_weights.mean * pred_bias,
         ]
         team_learner_variance = [
-            novelty_weights.variance * pred_novelty,
-            interest_weights.variance * pred_interest,
-            bias_weights.variance * pred_bias,
+            novelty_weights.variance * (pred_novelty**2),
+            interest_weights.variance * (pred_interest**2),
```

Comment:
What is the reasoning behind squaring the novelty and interest predictions?

Reply:
<img width="712" alt="image" src="https://github.com/TrueLearnAI/truelearn/assets/45459572/7ef738ba-cfce-4132-895d-3adbef4fd882">

Because these weights are Gaussian variables, when we multiply them with some constant, we need to multiply the variance by the square of the constant.